LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
is this license required? there is already a `LICENSE.txt` with the CPAL license